extensions/mssql/src/sqlClusterLookUp.ts

```diff
@@ -79,7 +79,10 @@ async function createSqlClusterConnInfo(sqlConnInfo: azdata.IConnectionProfile |
 	let endpoints: IEndpoint[] = serverInfo.options[constants.clusterEndpointsProperty];
 	if (!endpoints || endpoints.length === 0) { return undefined; }
 
-	let index = endpoints.findIndex(ep => ep.serviceName === constants.hadoopKnoxEndpointName);
+	let index = endpoints.findIndex(ep => ep.serviceName === constants.hadoopEndpointNameKnox);
+	if (index < 0) {
+		index = endpoints.findIndex(ep => ep.serviceName === constants.hadoopEndpointNameGateway);
```

Comment:
Let's do case-insensitive comparison please. Should avoid issues between 'knox' and 'Knox'.

Reply:
Helpfully it's already in the jupyterSessionManager code:
```
		return endpoints.find(ep => ep.serviceName.toLowerCase() === serviceName.toLowerCase());
```

Reply:
fixed